sshmux.go

```diff
@@ -88,30 +93,33 @@ func makeServer(config Config) (*Server, error) {
 		if loggerURL.Scheme == "udp" {
 			conn, err := net.Dial("udp", loggerURL.Host)
 			if err != nil {
-				log.Fatalf("Logger Dial failed: %s\n", err)
+				return nil, fmt.Errorf("logger dial failed: %s", err)
```

Comment:
We can probably change `%s` to `%w` if we're using `fmt.Errorf` with an `error`-type argument